### PR TITLE
Addition of R files to Fix CI Test Errors

### PR DESCRIPTION
**Pull request by Soumya624**
Added HpolytopeClass.R, SpectrahedronClass.R, VpolytopeClass.R, VpolytopeIntersectionClass.R and ZonotopeClass.R in the R directory that is solving the following:

Error: package or namespace load failed for ‘volesti’ in loadNamespace(package, lib.loc):
in package ‘volesti’ classes Zonotope, VpolytopeIntersection, Vpolytope, Spectrahedron, Hpolytope were specified for export but not defined.

### CODE REVIEW

File: .Rbuildignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 .gitingore
 LICENSE
 inst/checks
+^.*\.Rproj$
```
Comment (TolisChal) [inline .Rbuildignore:7]: Do we need these lines refering to `Rproj` folder? I think `Rproj` folder doesn't exist in this repo, right?

File: src/RcppExports.cpp
Comment (TolisChal) [inline src/RcppExports.cpp:1]: The file `RcppExports.cpp` is autogenerated during build time, isn't it?
If it is we need to delete it.

File: src/RcppExports.cpp [hunk 2]
```diff
@@ -264,7 +264,6 @@ BEGIN_RCPP
 END_RCPP
 }
 
-RcppExport SEXP _rcpp_module_boot_polytopes();
 RcppExport SEXP _rcpp_module_boot_spectrahedron();
```
Comment (TolisChal) [inline src/RcppExports.cpp:267]: Do we need to define the rcpp version of spectrahedron?
You have added an R class for spectrahedron, right?

I think you have to auto-generate again the RcppExports.cpp and update it in this PR.

File: .Rbuildignore (hunk 1 above)
Comment (Soumya624) [reply .Rbuildignore:7]: Hi @TolisChal ,

Thanks for this review. Updating in the latest PR.

**Review by TolisChal — APPROVED** [review]

**Comment by TolisChal** [comment]
Thanks for this PR! Everything looks good to me now. Do you have any additional changes to make in this PR>?

**Comment by Soumya624** [comment]
@TolisChal , 

Thanks for the review. At the moment, I don't any changes to add in this PR.

**Pull request merged**